frontend/src/views/auth/NavigationAuth.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <v-app-bar
+    v-if="$vuetify.breakpoint.mdAndUp"
+    app
+    clipped-left
+    color="blue-grey darken-4"
+    dark
+  >
+    <logo text />
+    <v-spacer />
+    <v-toolbar-items>
```

Comment:
you need to hide the button if the link is not configured (if we want to enable a not configured value)

Reply:
The termsOfServiceLinkTemplate has the same problem, and @manuelmeister made it consistent.
I opened another issue to fix this: https://github.com/ecamp/ecamp3/issues/4777